tracing-opentelemetry/src/main/java/io/micronaut/tracing/opentelemetry/instrument/util/TracingInvocationInstrumenterFactory.java

```diff
@@ -0,0 +1,38 @@
+/*
+ * Copyright 2017-2022 original authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.micronaut.tracing.opentelemetry.instrument.util;
+
+import io.micronaut.core.annotation.Experimental;
+import io.micronaut.core.annotation.Nullable;
+import io.micronaut.scheduling.instrument.InvocationInstrumenter;
+
+/**
+ * A factory interface for tracing invocation instrumentation.
+ * The factory method decides if instrumentation is needed.
+ *
+ * @author Denis Stepanov
+ */
+@Experimental
+public interface TracingInvocationInstrumenterFactory {
```

Comment:
Duplicate: https://github.com/micronaut-projects/micronaut-tracing/blob/master/tracing-core/src/main/java/io/micronaut/tracing/instrument/util/TracingInvocationInstrumenterFactory.java

Also should be deleted.